ci-operator/step-registry/gather/extra/gather-extra-commands.sh

```diff
@@ -33,6 +33,53 @@ then
     source "${SHARED_DIR}/proxy-conf.sh"
 fi
 
+echo "Collecting cluster data for analysis..."
+set +o errexit
+set +o pipefail
+if [ ! -f /tmp/jq ]; then
+  curl -L https://stedolan.github.io/jq/download/linux64/jq -o /tmp/jq && chmod +x /tmp/jq
```

Comment:
It's preferably to not curl anything from GitHub, because we often get throttled and it'll fail.  If `jq` is already available I would delete this.

If you do want to leave the curl, it should be for a specific sha with validation -- curling something without checking it's signature opens us up to potentially curling malware.

There's examples of how we do that for yq here:
https://github.com/openshift/release/blob/656c1d9385552dba5009ac4b1c442ae5f6740be9/ci-operator/step-registry/ipi/install/monitoringpvc/ipi-install-monitoringpvc-commands.sh#L15-L23

Reply:
If you do want to leave the curl, it should be for a specific sha with validation -- curling something without checking it's signature opens us up to potentially curling malware.

There's examples of how we do that for yq here:
https://github.com/openshift/release/blob/656c1d9385552dba5009ac4b1c442ae5f6740be9/ci-operator/step-registry/ipi/install/monitoringpvc/ipi-install-monitoringpvc-commands.sh#L15-L23